README.md

```diff
@@ -5,3 +5,217 @@
 [![Build Status](https://dev.azure.com/trustbloc/edge/_apis/build/status/trustbloc.edge-service?branchName=master)](https://dev.azure.com/trustbloc/edge/_build/latest?definitionId=27&branchName=master)
 [![codecov](https://codecov.io/gh/trustbloc/edge-service/branch/master/graph/badge.svg)](https://codecov.io/gh/trustbloc/edge-service)
 [![Go Report Card](https://goreportcard.com/badge/github.com/trustbloc/edge-service)](https://goreportcard.com/report/github.com/trustbloc/edge-service)
+
+## Prerequisites (for running tests and demos)
+- Go 1.13
+- Docker
+- Docker-Compose
+- Make
+
+## Targets
+```
+# run checks and unit tests
+make all
+# run linter checks
+make checks
+# run unit tests
+make unit-test
+```
+
+#Dependency
+- [EDV](https://github.com/trustbloc/edv)
+
+# edge-service
```

Comment:
move it to subfolders like https://github.com/trustbloc/edge-sandbox/tree/master/docs